features/steps/steps.py

```diff
@@ -88,6 +88,17 @@ def __str__(self):
         else:
             return f"No instances of type {self.type_name} were encountered"
 
+@dataclass
+class representation_value_error:
+    inst: ifcopenshell.entity_instance
+    duplicate_value: str
+    duplicate_representations: ifcopenshell.entity_instance
```

Comment:
type should be `list` or, if you want to annotate also the element, sth like `typing.Sequence[ifcopenshell.entity_instance]`

Reply:
This class is deleted, but I think it is the same for the other class. Updated in https://github.com/buildingSMART/ifc-gherkin-rules/commit/44652325ccf0529efbee4f32a25e7b7f56650243
I took this approach from an already existing rule (SPS001): https://github.com/buildingSMART/ifc-gherkin-rules/blob/a714b1fe1d248d1e9088db3432d9d51487c01331/features/steps/steps.py#L80-L89 
To my understanding this should then also be changed, as well as type specifications dataclasses for other rules. But I think it would be better to do this in the non-rule sprint and not in this PR to avoid overlap.